book/src/building-from-source.md

```diff
@@ -145,8 +145,8 @@ you can configure Helix to show up in the application menu by copying the
 provided `.desktop` and icon files to their correct folders:
 
 ```sh
-cp contrib/Helix.desktop ~/.local/share/applications
-cp contrib/helix.png ~/.icons # or ~/.local/share/icons
+cp contrib/Helix.desktop -- "$HOME/.local/share/applications"
```

Comment:
This should be equivalent to the `~` expansion, no?

Reply:
For most shells, yes. But [`HOME` is more portable](https://stackoverflow.com/questions/11587343/difference-between-home-and-tilde/11587382#11587382)